t/22-dashboard.t

```diff
@@ -206,6 +206,11 @@ is($get->tx->res->dom->find('#review-' . $test_parent->id . '-0048')->size,
     1, 'review badge for build 0048 shown, despite unreviewed softfails');
 is($get->tx->res->dom->find('#child-review-' . $test_parent->id . '-0048')->size,
     1, 'review badge for build 0048 shown on child-level, despite unreviewed softfails');
+# mark one softfailed job also with a bugref which should keep the reviewed badge
+$opensuse_group->jobs->find({id => 99939})->comments->create({text => 'poo#4322', user_id => 99901});
+$get = $t->get_ok('/?limit_builds=20')->status_is(200);
+is($get->tx->res->dom->find('#child-review-' . $test_parent->id . '-0048')->size,
+    1, 'review badge still there after labelling softfails');
```

Comment:
Think this could also be written as `$t->element_count_is("#child-review-@{[$test_parent->id]}-0048", 1, 'review badge still there after labelling softfails')` (and appended as a method call to the previous line).